invenio_cli/commands/containers.py

```diff
@@ -218,9 +218,12 @@ def rdm_fixtures(self, project_shortname):
 
     def translations(self, project_shortname):
         """Steps to compile translations for the instance."""
+        print(self.cli_config.get_project_dir())
```

Comment:
local project path

Reply:
It is the local path, but it could be `None`. The `TranslationsCommands` requires it as parameter in the constructor, but then there is a `directory or ...` which means it won't be used. TL;DR we can potentially make it optional, or play with inheritance and make it optional in a subclass `TranslationContainerizedcommands`